integreat_cms/cms/views/utils/hix.py

```diff
@@ -59,13 +62,21 @@ def get_hix_score(request, region_slug):
     # Don't pass texts larger than 100kb to the api in order to avoid being vulnerable to dos attacks
     if len(request.body) > MAX_TEXT_LENGTH:
         return JsonResponse({"error": "Request too large"})
+
     body = json.loads(request.body.decode("utf-8"))
     text = body["text"]
 
     if not isinstance(text, str) or not text.strip():
         logger.warning("Received invalid text: %r", text)
         return JsonResponse({"error": f"Invalid text: '{text}'"})
 
+    # Replace line break \n with \r\n
```

Comment:
```suggestion
    # Normalize text to get same HIX result as with form submission
```
I think it's pretty obvious that the following line replaces `\n` with `\r\n`, so no need to duplicate this information in a comment.
We could however explain why we're doing it.